src/main/features/first-contact/routes/claim-summary.ts

```diff
@@ -7,6 +7,7 @@ import { ClaimReferenceMatchesGuard } from 'first-contact/guards/claimReferenceM
 import { JwtExtractor } from 'idam/jwtExtractor'
 import { ClaimantRequestedCCJGuard } from 'first-contact/guards/claimantRequestedCCJGuard'
 import { OAuthHelper } from 'idam/oAuthHelper'
+import { getInterestDetails } from '../../../common/interest'
```

Comment:
Please change this to an absolute import.

Reply:
done, intelliJ auto imports it using all the dots. Need to train myself to check for it in future.